lib/internal/process/next_tick.js

```diff
@@ -19,6 +19,69 @@ function setupNextTick() {
   const { kInit, kDestroy, kAsyncIdCounter } = async_wrap.constants;
   const { async_id_symbol, trigger_async_id_symbol } = async_wrap;
 
+  class SmallQueue {
+    constructor() {
+      this.list = [];
+      this.top = 0;
+      this.btm = 0;
+    }
+
+    push(tick) {
```

Comment:
Nit: I’d spell out `tickInfo` here, I think that’s what these objects are called elsewhere